test/cppsuite/test_harness/random_generator.h

```diff
@@ -41,14 +40,12 @@ class random_generator {
         _distribution = std::uniform_int_distribution<>(0, _characters.size() - 1);
     }
 
-    static random_generator *_instance;
     std::mt19937 _generator;
```

Comment:
Can these fields be const as well?

Reply:
I don't think so as they are created later on, what did you have in mind ?